submissions/nadsatt/friends-app/modules/icon-list.js

```diff
@@ -0,0 +1,63 @@
+import { Icon } from './icon-item.js';
+
+export class IconList{
```

Comment:
This logic should be in filter - filter itself should know when it needs to be opened

Reply:
![image](https://user-images.githubusercontent.com/2997359/104048423-d9e8a900-51eb-11eb-98b4-4f20ad728a75.png)

This should be one class - `SortList` and it should contain `AgeSort`, `NameSort`, etc.

When you click on icon Age, it should call method `open` of `AgeSort` and `AgeSort` should change it's own icon in a way as he knows.